classes/03class/exercises/c03-docker01/ramiray/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Setup build arguments with terraform version=0.12.20
+ARG TERRAFORM_VERSION=0.12.20
+
+# Download and install Terraform binary
+FROM alpine:latest
```

Comment:
using `latest` is not a good practice.
Always use a specific version to avoid having changes upstream affecting your work.